src/pages/Winners2021.js

```diff
@@ -0,0 +1,369 @@
+import React from "react"
+import s from "styled-components"
+import { Title, Subtitle, Text, Link } from "../components/shared/Typography"
+import { STEEL_BLUE, GREEN } from "../utils/constants"
+import Renecycle from '../images/Renecycle.jpg'
+import Hunter from '../images/Hunter Lee _ CEO.jpg'
+import Mina from '../images/Mina Liang _ COO.jpg'
+import Henry from '../images/Henry Lei.jpeg'
+import David from '../images/David Chang.jpeg'
+import Ian from '../images/Ian Gordon.jpeg'
+import Folia from '../images/Folia.jpg'
+import Rosanna from '../images/RosannaHeadshot.jpg'
+import Caroline from '../images/CarolineHeadshot.jpeg'
+import Zachary from '../images/ZacharyHeadshot.jpeg'
+import Stephanie from '../images/StephanieHeadshot.jpg'
+import Ryoma from '../images/RyomaHeadshot.jpg'
+import TeamChaeto from '../images/TeamChaeto.jpg'
+import ChrisNick from '../images/Team Chaeto.jpeg'
+import NASADYA from '../images/NASADYA.jpg'
+import Chaitanya from '../images/Chaitanya_Gulati Headshot.png'
+import Praneeth from '../images/NASADYA_Praneeth Nagarjuna_Headshot.jpeg'
+import Devansh from '../images/NASADYA_Devansh-Sethia_Headshot.jpg'
+import Khadeeja from '../images/NASADYA_Khadeeja-Malik_Headshot_1.jpg'
+import Dan from '../images/NASADYA_Dan Duffy_Headshot.jpg'
+import Nate from '../images/Nate-Headshot_1.jpg'
+import Madhu from '../images/NASADYA_MadhuDevi.jpg'
```

Comment:
again, you probably could benefit a lot by splitting the images dir into more folders and standardizing image names.

Reply:
This is a LOT of images, and this file is really chonky. Might be good to split the file into multiple files per team for clarity! Might even be possible to extract out a "Team" component

Reply:
Prefer png over jpg when possible

Also can you choose whether you like jpeg or jpg?